ktsune/safe-space-fe
# Setup check in

#### What’s this PR do?
This PR restructures the CheckInForm so that it is a component that renders 3 other components (BasicInfoForm, NeedsForm, EmergencyContactForm).
#### Where should the reviewer start?
The reviewer should begin at the CheckInForm js file, and then review the newly added components.
#### How should this be manually tested?
Currently, to test this add-in, the reviewer should start at the login page, click on a center, click submit, then click the CheckInForm information to see what is displayed on the DOM.
